src/main/resources/messages.properties

```diff
@@ -0,0 +1,5 @@
+validation.title.notempty=Title is required
+validation.author.notempty=Author is required
+validation.isbn.notempty=The ISBN is required
+validation.isbn.valid=The ISBN is not valid
+validation.price.valid=The price must be greater than 0
```

Comment:
There is no reason to put it to separate file, It is ok when messages is just a text inside annotations